l10n_it_edi_related_document_type/models/account_move.py

```diff
@@ -0,0 +1,114 @@
+from odoo import fields, models
+
+from odoo.addons.l10n_it_edi.models.account_move import get_text
+
+RELATED_DOCUMENT_TYPES = {
+    "order": "DatiOrdineAcquisto",
+    "contract": "DatiContratto",
+    "agreement": "DatiConvenzione",
+    "reception": "DatiRicezione",
+    "invoice": "DatiFattureCollegate",
+}
+
+
+class AccountMoveRelatedDocumentType(models.Model):
+    _name = "account.move.related_document_type"
+    _description = "E-invoice Related Document Type"
```

Comment:
```suggestion
    _name = "account.move.related_document"
    _description = "E-invoice Related Document"
```
Il modello è per i documenti correlati, che sono di diversi tipi a seconda del campo `type`; qui invece stai dichiarando un modello che sarebbe un "Tipo di documento correlato", che a sua volta ha un tipo..?

Reply:
:+1: